src/main/java/sirius/biz/codelists/LookupValueSuggestionController.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.codelists;
+
+import sirius.biz.web.BizController;
+import sirius.kernel.di.std.Part;
+import sirius.kernel.di.std.Register;
+import sirius.web.controller.AutocompleteHelper;
+import sirius.web.controller.Routed;
+import sirius.web.http.WebContext;
+import sirius.web.security.LoginRequired;
+
+/**
+ * Endpoint for autocomplete suggestions for {@link LookupValue} and {@link LookupValues} fields
```

Comment:
Optional: Start with a verb as `Provides an endpoint` or `Provides autocomplete suggestions for ...`